src/WorkoutsMain/WorkoutsMain.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import './WorkoutsMain.css';
+
+export default function WorkoutsMain() {
+    return (
+        <div className="WorkoutsMain">
+            <main className="container">
+                <div className="container-content">Workout Routine</div>
+                <div className="container-content">Workout Routine</div>
+                <div className="container-content">Workout Routine</div>
+                <div className="container-content">Workout Routine</div>
+                <div className="container-content">Workout Routine</div>
+                <div className="container-content">Workout Routine</div>
+            </main>
+        </div>
+    )
```

Comment:
I would add a semicolon at the end of the line just to keep it consistent with App.js